.travis.yml

```diff
@@ -11,4 +11,7 @@ before_install:
   - sudo apt-get install -qq gcc-arm-none-eabi
   - mkdir -p build/apps
 
-script: make build/main.elf
+script:
+  - make build/main.elf
+  - make clean-all
+  - make PLATFORM=nrf_pca10001 CHIP=nrf51822 APPS=c_blinky
```

Comment:
Oh great! So this will test compilation for both platforms. I think `APPS=c_blinky` might be somewhat extraneous since it just gets compiled to a blob and linked -- there is nothing really specific to this platform here.

But no matter, it's at least a place for now to document how to get an example that actually runs on the board.